config/monitoring.config

```diff
@@ -0,0 +1,113 @@
+[
+  { kernel,
+    [
+     {inet_dist_use_interface, {127,0,0,1}}
+    ]
+  },
+
+  { aehttp, [
+      {external, [
+          {acceptors, 10},
+          {port, 3013}
+          ]},
+      {internal, [
+          {acceptors, 10},
+          {port, 3113},
+          {debug_endpoints, true}
+          ]},
+      {channel, [
+          {websocket, [ {port, 3014},
+                        {handlers, 100},
+                        {listen_address, <<"0.0.0.0">>}
+                      ]}
+          ]}
+  ]},
+
+  {aecore, [
+      {peers, []},
+      {peer_password, <<"secret">>},
+      {persist, false},
+      {aec_mining, {29, [{<<"mean29-generic">>, <<"-t 1">>, false, 1, undefined, <<"aecuckoo">>}]}}
+    ]
+  },
+
+  {jobs,
+   [{queues,
+     [{sync_ping_workers, [{regulators, [{counter, [{limit, 3}]}]}]},
+      {sync_task_workers, [{regulators, [{counter, [{limit, 10}]}]}]},
+      {sync_gossip_workers, [{regulators, [{counter, [{limit, 10}]}]}]}
+     ]}
+   ]},
+
+  {lager, [
+      {error_logger_flush_queue, false},
+      {handlers, [
+          {lager_console_backend, [{level, info}]},
+          {lager_file_backend,
+            [{file, "log/aeternity.log"}, {level, debug},
+             {size, 4194303}, {date, "$D0"}, {count, 10}]}
+      ]},
+      {extra_sinks, [
+           {epoch_mining_lager_event, [
+             {handlers, [
+               {lager_file_backend, [
+                  {file, "log/aeternity_mining.log"},
+                  {level, info},
+                  {size, 4194303}, {date, "$D0"}, {count, 3}
+               ]}
+             ]}
+           ]},
+           {epoch_metrics_lager_event, [
+             {handlers, [
+               {lager_file_backend, [
+                  {file, "log/aeternity_metrics.log"},
+                  {level, info},
+                  {size, 4194303}, {date, "$D0"}, {count, 3},
+                  {formatter, lager_default_formatter},
+                  {formatter_config, [time, " ", message, "\n"]}
+               ]}
+             ]}
+           ]},
+           {aeminer_lager_event, [
+             {handlers, [
+	           {lager_file_backend, [
+	              {file, "log/aeternity_pow_cuckoo.log"},
+	              {level, debug},
+	              {size, 4194303}, {date, "$D0"}, {count, 3}
+	           ]}
+	         ]}
+	       ]},
+           {epoch_sync_lager_event, [
+             {handlers, [
+                {lager_console_backend, [{level, info}]},
+                {lager_file_backend, [
+                  {file, "log/aeternity_sync.log"}, {level, debug},
+                  {size, 4194303}, {date, "$D0"}, {count, 3}
+               ]}
+             ]}
+           ]}
+      ]}
+  ]},
+
+  {sasl, [
+    {sasl_error_logger, false}
+  ]},
+
+  {mnesia, [{dir, "data/mnesia"}]},
+
+  {setup, [
+           {abort_on_error, true},
+           {data_dir, "data"},
+           {log_dir, "log"}
+          ]},
+
+  {aemon, [
```

Comment:
I see this is dev.config with
```
{aemon, [
  {active, true},
  {publisher_pubkey, <<>>},
  {publisher_privkey, <<>>},
  {publisher_spend_amount, 20000},
  {publisher_autostart, true},
  {publisher_interval, 10000},
  {publisher_spend_ttl, 10}
]}
```
added. Is this file unused? Who picks it?

Reply:
If unused, I expect it shall be deleted.

Reply:
removed